doc/api/errors.md

```diff
@@ -970,172 +962,166 @@ made to mark a stream and dependent of itself.
+`https:`.
 
 <a id="ERR_INDEX_OUT_OF_RANGE"></a>
 ### ERR_INDEX_OUT_OF_RANGE
 
-Used when a given index is out of the accepted range (e.g. negative offsets).
+A given index is out of the accepted range (e.g. negative offsets).
 
 <a id="ERR_INSPECTOR_ALREADY_CONNECTED"></a>
 ### ERR_INSPECTOR_ALREADY_CONNECTED
 
-When using the `inspector` module, the `ERR_INSPECTOR_ALREADY_CONNECTED` error
-code is used when an attempt is made to connect when the inspector is already
-connected.
+While using the `inspector` module, an attempt was made to connect when the
+inspector is already connected.
 
 <a id="ERR_INSPECTOR_CLOSED"></a>
 ### ERR_INSPECTOR_CLOSED
 
-When using the `inspector` module, the `ERR_INSPECTOR_CLOSED` error code is
-used when an attempt is made to use the inspector after the session has
-already closed.
+While using the `inspector` module, an attempt was made to use the inspector
+after the session has already closed.
 
 <a id="ERR_INSPECTOR_NOT_AVAILABLE"></a>
 ### ERR_INSPECTOR_NOT_AVAILABLE
 
-Used to identify when the `inspector` module is not available for use.
+The `inspector` module is not available for use.
 
 <a id="ERR_INSPECTOR_NOT_CONNECTED"></a>
 ### ERR_INSPECTOR_NOT_CONNECTED
 
-When using the `inspector` module, the `ERR_INSPECTOR_NOT_CONNECTED` error code
-is used when an attempt is made to use the inspector before it is connected.
+While using the `inspector` module, an attempt was made to use the inspector
+before it was connected.
 
 <a id="ERR_INVALID_ARG_TYPE"></a>
 ### ERR_INVALID_ARG_TYPE
 
-Used generically to identify that an argument of the wrong type has been passed
-to a Node.js API.
+An argument of the wrong type has been passed to a Node.js API.
 
 <a id="ERR_INVALID_ARG_VALUE"></a>
 ### ERR_INVALID_ARG_VALUE
 
-Used generically to identify that an invalid or unsupported value has been
-passed for a given argument.
+An invalid or unsupported value has been passed for a given argument.
 
 <a id="ERR_INVALID_ARRAY_LENGTH"></a>
 ### ERR_INVALID_ARRAY_LENGTH
 
-Used when an Array is not of the expected length or in a valid range.
+An Array is not of the expected length or in a valid range.
 
 <a id="ERR_INVALID_ASYNC_ID"></a>
 ### ERR_INVALID_ASYNC_ID
 
-Used with `AsyncHooks` when an invalid `asyncId` or `triggerAsyncId` is passed.
-An id less than -1 should never happen.
+An invalid `asyncId` or `triggerAsyncId` was passed using `AsyncHooks`. An id
+less than -1 should
+never happen.
 
 <a id="ERR_INVALID_BUFFER_SIZE"></a>
 ### ERR_INVALID_BUFFER_SIZE
 
-Used when performing a swap on a `Buffer` but it's size is not compatible with the operation.
+A swap was performed on a `Buffer` but its size is not compatible with the
+operation.
 
 <a id="ERR_INVALID_CALLBACK"></a>
 ### ERR_INVALID_CALLBACK
 
-Used generically to identify that a callback function is required and has not
-been provided to a Node.js API.
+A callback function is required and has not been provided to a Node.js API.
```

Comment:
Maybe `A callback function was required but had not been provided to a Node.js API.`